pkg/scheduler/testing/wrappers.go

```diff
@@ -415,6 +416,8 @@ func (p *PodWrapper) Req(resMap map[v1.ResourceName]string) *PodWrapper {
 		res[k] = resource.MustParse(v)
 	}
 	p.Spec.Containers = append(p.Spec.Containers, v1.Container{
+		Name:  fmt.Sprintf("con%d", len(p.Spec.Containers)),
+		Image: imageutils.GetPauseImageName(),
```

Comment:
For UTs, we don't need to set the image name; for integration tests, we need to set a name (fake one is fine). However given this may be used by e2e as well, probably better to set a non-fake image name here.